src/Console/Commands/ShortenCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class ShortenCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'shorten';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Shorten a given URL';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $driver = $this->laravel['url.shortener']->driver($this->input->getOption('driver'));
+
+        $shortUrl = $driver->shorten($this->input->getArgument('url'));
+
+        $this->info('URL shortened successfully.');
+        $this->info("Your short URL is: $shortUrl");
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getArguments()
+    {
+        return [
+            ['url', InputArgument::REQUIRED, 'The URL to shorten'],
+        ];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getOptions()
```

Comment:
Why are you using this? You could easily (and I would prefer this) define options and arguments in the Laravel way